.github/workflows/build.yml

```diff
@@ -28,11 +28,11 @@ jobs:
   test:
     strategy:
       matrix:
-        go: ["1.20.x", "1.19.x"]
+        go: ["1.21.x", "1.20.x"]
```

Comment:
are we missing `1.19.x` here as well? Looks like we set that for installing modules and for the `dist` strategy below

Reply:
This is fixed